web/src/components/Cart/Cart.js

```diff
@@ -94,12 +95,24 @@ const ShoppingCartButton = styled(Button)`
     `}
 `
 
-const CartItem = ({ image, quantity, name }) => {
+const RemoveCartButton = styled(Button)`
+  background-color: var(--red-7);
+
+  &:hover {
+    background-color: var(--red-6);
+    transition: background-color 200ms;
+  }
+`
+const CartItem = ({ image, quantity, name, id }) => {
+  const removeFromCart = useRemoveFromCart()
   return (
-    <Row>
+    <Row style={{ width: '100%' }}>
       <Quantity>{quantity}</Quantity>
       <CartImage src={image} />
       <p style={{ fontSize: 'calc(var(--font-size-1) / 1.125)' }}>{name}</p>
+      <RemoveCartButton style={{ marginLeft: 'auto' }}>
+        <Trash onClick={() => removeFromCart({ id })} />
```

Comment:
The SVG is a little big; I tried a few sizes and `--size-4` feels right (also updated the icon, as per another comment):
```suggestion
         <Trash2 style={{ width: 'var(--size-4)' }} />
```

Reply:
updated!